wdl/AnnotateVcf.wdl

```diff
@@ -1,113 +1,149 @@
 version 1.0
 
-import "AnnotateFunctionalConsequences.wdl" as func
-import "PruneAndAddVafs.wdl" as pav
-import "AnnotateExternalAF.wdl" as eaf
+import "Structs.wdl"
+import "ShardedAnnotateVcf.wdl" as sharded_annotate_vcf
+import "TasksMakeCohortVcf.wdl" as MiniTasks
+import "HailMerge.wdl" as HailMerge
 
 workflow AnnotateVcf {
 
   input {
-    File vcf
-    File vcf_idx
+    Array[File] vcf_list  # Must be either single full VCF (array of length 1) or array of VCFs sharded by contig. Outputs will match
+    Array[File]? vcf_idx_list
     File contig_list
-    String prefix
+    Array[String] prefix_list
```

Comment:
This design works but it requires some complicated code below. I think it would be better to forego the assumption that it's every sharded by contig. Looking through, the contig is only used for file prefixes, so we could try something a little more general that would be simpler to implement.

I'd suggest dropping the `contig_list` input and using `basename` on the VCFs to generate the `prefix_list`. If it's sharded by contig, we'll still get the contig labels on all the outputs. This way we don't need to worry about whether we're in the sharded or unsharded case nor whether the contig list length matches the array length.